app/services/moves/updater.rb

```diff
@@ -55,6 +53,9 @@ def attributes
 
       attributes[:profile] = Profile.find_by(id: profile_attributes.dig(:data, :id)) if profile_attributes.present?
 
+      profile = attributes[:profile] || move.profile
+      profile.documents = Document.where(id: document_ids) unless document_attributes.nil?
```

Comment:
```suggestion
      profile.documents = Document.where(id: document_ids) if document_attributes.present?
```

Reply:
Can't do that because nil and empty [] mean specific things (see specs)

Reply:
Alessandro suggested generalising pulling out relationships which is a great idea.